src/components/EpicButtons.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { css } from 'emotion';
+import { space } from '@guardian/src-foundations';
+import { Button } from './Button';
+import { EpicTracking } from './ContributionsEpicTypes';
+import { Cta, Variant } from '../lib/variants';
+import { addTrackingParams } from '../lib/tracking';
+import { addRegionIdToSupportUrl } from '../lib/geolocation';
+
+const buttonWrapperStyles = css`
+    margin: ${space[6]}px ${space[2]}px ${space[1]}px 0;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+
+    &.hidden {
+        display: none;
+    }
+`;
+
+const paymentImageStyles = css`
+    display: inline-block;
+    width: auto;
+    height: 25px;
+    margin: ${space[1]}px 0;
+`;
+
+const buttonMargins = css`
+    margin: ${space[1]}px ${space[2]}px ${space[1]}px 0;
+`;
+
+const PrimaryCtaButton = ({
+    cta,
+    tracking,
+    countryCode,
+}: {
+    cta?: Cta;
+    tracking: EpicTracking;
+    countryCode?: string;
+}): JSX.Element | null => {
+    if (cta) {
+        const buttonText = cta.text || 'Support The Guardian';
+        const baseUrl = cta.baseUrl || 'https://support.theguardian.com/contribute';
+        const urlWithRegion = addRegionIdToSupportUrl(baseUrl, countryCode);
+        const urlWithRegionAndTracking = addTrackingParams(urlWithRegion, tracking);
+
+        return (
+            <div className={buttonMargins}>
+                <Button onClickAction={urlWithRegionAndTracking} showArrow>
+                    {buttonText}
+                </Button>
+            </div>
+        );
+    } else {
+        return null;
+    }
+};
+
+export const EpicButtons = ({
+    variant,
+    tracking,
+    countryCode,
+}: {
+    variant: Variant;
+    tracking: EpicTracking;
+    countryCode?: string;
+}): JSX.Element | null => {
+    const { cta, secondaryCta, showReminderFields } = variant;
+    if (!cta) {
+        return null;
+    }
+
+    return (
+        <div className={buttonWrapperStyles} data-target="epic-buttons" data-testid="epic=buttons">
+            <PrimaryCtaButton cta={cta} tracking={tracking} countryCode={countryCode} />
+
+            {secondaryCta && secondaryCta.baseUrl && secondaryCta.text && (
```

Comment:
I suggest a little tweak to handle that use case we talked about doing separately:
```suggestion
            {secondaryCta && secondaryCta.baseUrl && secondaryCta.text && ! showReminderFields && (
```
This will omit the secondary CTA if there's a reminder, assuming we want to show either one or the other, prioritising the reminder fields.

Reply:
Hey cool. It looks like it's the secondary button which takes priority over the reminder. If it sounds okay I'm going to stick with following up in a separate PR to achieve this (and add some tests for this too).

Reply:
Cool 👌